stolon/templates/keeper-statefulset.yaml

```diff
@@ -3,25 +3,24 @@ kind: StatefulSet
 metadata:
   name: {{ template "stolon.keeper.fullname" . }}
   labels:
-    app: {{ template "stolon.keeper.fullname" . }}
-    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    app: "stolon-keeper"
```

Comment:
I suppose this should be  `app: {{ template "stolon.keeper.fullname" . }}`

Reply:
Yes